README.md

```diff
@@ -1,26 +1,110 @@
+           <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032351-9d9d5619-8ef7-470a-9eec-2744ece54553.png" />
+       </picture>
+   </a>&nbsp;&nbsp;&nbsp;
+   <a href="https://t.me/app_revanced">
+      <picture>
+         <source height="24px" media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/13122796/178032213-faf25ab8-0bc3-4a94-a730-b524c96df124.png" />
+         <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032213-faf25ab8-0bc3-4a94-a730-b524c96df124.png" />
+      </picture>
+   </a>&nbsp;&nbsp;&nbsp;
+   <a href="https://x.com/revancedapp">
+      <picture>
+         <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/93124920/270180600-7c1b38bf-889b-4d68-bd5e-b9d86f91421a.png">
+         <img height="24px" src="https://user-images.githubusercontent.com/93124920/270108715-d80743fa-b330-4809-b1e6-79fbdc60d09c.png" />
+      </picture>
+   </a>&nbsp;&nbsp;&nbsp;
+   <a href="https://www.youtube.com/@ReVanced">
+      <picture>
+         <source height="24px" media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/13122796/178032714-c51c7492-0666-44ac-99c2-f003a695ab50.png" />
+         <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032714-c51c7492-0666-44ac-99c2-f003a695ab50.png" />
+     </picture>
+   </a>
+   <br>
+   <br>
+   Continuing the legacy of Vanced
+</p>
 
-The official ReVanced Website.
+# 🌐 ReVanced Website
 
-## Developing
+![Deployment status](https://img.shields.io/github/actions/workflow/status/ReVanced/revanced-website/deploy.yml)
+![GPLv3 License](https://img.shields.io/badge/License-GPL%20v3-yellow.svg)
 
-Install dependencies with `npm install` (or `pnpm install`).
+This repository housed the content of ReVanced Website.
 
-Start a development server by running:
+## ❓ About
 
-```bash
-npm run dev
+Website is a site built from Svelte that uses ReVanced API to quickly provides 
+way to retrieve information regarding announcement, patches, contributor, and much more.
 
-# or start the server and open the app in a new browser tab
-npm run dev -- --open
-```
+## 💪 Features
 
-## Building
+Some of the features the website provide are:
 
-To create a production build, run:
+* ⚙️ **Powered by ReVanced API**: Utilize most of the features provided by [ReVanced API](https://github.com/ReVanced/revanced-api)
+* 📢 **Announcement**: Easy way to create and view announcements to users on the internet
+* ♿ **Accessibility-friendly**: Accessible by anyone who needs an accessibility feature
+* 🔎 **Patch viewer**: List all the patches available in [ReVanced Patches](https://github.com/ReVanced/revanced-patches)
+* ✨ **And much more!**
 
-```bash
-npm run build
-```
+## 🚀 How to get started
 
-You can preview the production build with `npm run preview`.
+To use ReVanced Website, go to https://revanced.app
```

Comment:
This section should show setup & build instructions (.env setup & build command)